tests/test-sources/plugins/by-name/grug-far/default.nix

```
@@ -0,0 +1,37 @@
+{
+  empty = {
+    plugins.grug-far.enable = true;
+  };
+
+  # Just a small subset of the settings
+  defaults = {
+    plugins.grug-far = {
+      enable = true;
+      settings = {
```

```suggestion
      enable = true;

      settings = {
```

@GaetanLepage would make this suggestion. Personally I don't care too much, but I guess we should be consistent :joy:

---

> @GaetanLepage would make this suggestion.

I would indeed.
ADD THIS SPACE.

NOW.
Or I'm blocking this PR

---

(And please, declare all the 420 options of this plugin with proper types and sub-element-rawtype-support, or this doesn't get merged.)